src/lib/BytesHelper.ts

```diff
@@ -25,8 +25,8 @@ export function argToBytes(
     v = val.toFixed();
   }
 
-  return hexToBytes(
-    padLeft(toHex(v), 64, '0'),
+  return Web3.utils.hexToBytes(
+    Web3.utils.padLeft(Web3.utils.toHex(v), 64, '0'),
```

Comment:
Actually may have to do the same thing I did above (add `.map(x => [x])`) to return `number[][]`
Not sure why this didn't cause an error.

Annoying how the contract types take in a `number[][]` for bytes arrays. It makes no sense.